### DISCUSSION TITLE
Piggy Banks: Suggested per month

### DISCUSSION THREAD

**Discussion by macuros** [firefly-iii/firefly-iii#5271] [Firefly III] [answered]
Hello,

How this actually works? How the suggested monthly amount is calculated?

For instance, I've set a piggy bank for 110 € . Tha target date is 11th January 2022. I had 90 € already inside. Today I went to fill the piggy banks and that one suggested to fill 20 €, so 100%. But there are still 2 months till January. 

Just trying to understand how it works. Piggy banks (goals) are very important to me in finance manager. That is the main thing why I use them.

Thank you.

**Comment by JC5** [comment, marked as answer]
Root cause is a bug in PHP which I hope will be fixed soon: https://3v4l.org/uH88b

In most cases it works correctly.
